doc/release-process.md

```diff
@@ -29,7 +29,7 @@ Before every major release:
 
 ### First time / New builders
 
-If you're using the automated script (found in [contrib/gitian-build.sh](/contrib/gitian-build.sh)), then at this point you should run it with the "--setup" command. Otherwise ignore this.
+If you're using the automated script (found in [contrib/gitian-build.py](/contrib/gitian-build.py)), then at this point you should run it with the "--setup" command. Otherwise ignore this.
```

Comment:
Could you please also fix this in `contrib/README.md` and https://github.com/bitcoin-core/docs/blob/master/gitian-building.md#initial-gitian-setup

Reply:
@MarcoFalke sure can

Reply:
@MarcoFalke both now done